Common.Logging.Scopes/Common.Logging.Scopes.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Why NET.Sdk? specially since Common.Logging.Scopes.Tests is a .netfx std project?!

Reply:
Resolved issue on last comment; pending to move Test project from .NET Framework 4.5 to SDK with target net45.